src/app/_components/footer.tsx

```diff
@@ -36,9 +36,7 @@ export function Footer() {
 				<div className="text-center mt-5">
 					<hr className="border-t-2 border-gray-400" />
 					<p className="text-sm text-gray-400 mt-3">
-						<span className="mr-2">
-							&copy; {currentYear} SMSWithoutBorders. All rights reserved.
-						</span>
+						<span className="mr-2">&copy; {currentYear} SMSWithoutBorders.</span>
```

Comment:
We can safely remove `className="mr-2"`, as it is not being used.

```suggestion
						<span>&copy; {currentYear} SMSWithoutBorders.</span>
```